### PR TITLE
feat: add admin home

### PR DESCRIPTION
**Pull request by lionelB**
- create a deployment (and basic admin stat)

### CODE REVIEW

File: src/lib/ui/AdminDeployment/AdminDeploymentCreateLayer.svelte [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		accounts: {
+			data: {
+				username: '',
+				type: 'manager',
+			},
+		},
+	};
+
+	async function handleSubmit() {
+		console.log('submit');
```
Comment (Augustin82) [inline src/lib/ui/AdminDeployment/AdminDeploymentCreateLayer.svelte:28]: nitpick : remove this?

File: src/lib/ui/AdminDeployment/AdminDeploymentCreateLayer.svelte [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<script lang="ts">
+	import { CreateDeploymentDocument } from '$lib/graphql/_gen/typed-document-nodes';
+	import { openComponent } from '$lib/stores';
+
+	import { mutation, operationStore } from '@urql/svelte';
+	import { Input } from '../base';
```
Comment (Augustin82) [inline src/lib/ui/AdminDeployment/AdminDeploymentCreateLayer.svelte:6]: nitpick : use $lib alias?

File: src/lib/ui/AdminDeployment/AdminDeploymentCreateLayer.svelte [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	import Button from '../base/Button.svelte';
```
Comment (Augustin82) [inline src/lib/ui/AdminDeployment/AdminDeploymentCreateLayer.svelte:7]: nitpick : use $lib alias?

File: src/lib/ui/AdminDeployment/AdminDeploymentDetail.svelte [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+	type ProfessionalAggregateSub = Pick<ProfessionalAggregate, 'aggregate'>;
+
+	export let professional_aggregate: ProfessionalAggregateSub;
+</script>
+
+<h1 class="fr-h2">
+	Déploiement <span class="text-france-blue-500">{deployment?.label}</span>
+</h1>
+<p>Référent: {deployment?.managers.map((item) => displayFullName(item)).join(', ')}</p>
```
Comment (Augustin82) [inline src/lib/ui/AdminDeployment/AdminDeploymentDetail.svelte:29]: nitpick : use non-breaking space before the ":"?

File: src/lib/ui/AdminDeployment/AdminDeploymentDetail.svelte [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+		beneficiaries_aggregate: BeneficiariesAggregateSub;
+	};
+
+	type ProfessionalAggregateSub = Pick<ProfessionalAggregate, 'aggregate'>;
+
+	export let professional_aggregate: ProfessionalAggregateSub;
+</script>
+
+<h1 class="fr-h2">
+	Déploiement <span class="text-france-blue-500">{deployment?.label}</span>
```
Comment (Augustin82) [inline src/lib/ui/AdminDeployment/AdminDeploymentDetail.svelte:27]: nitpick : use \<Text\> to handle null

File: src/lib/ui/AdminDeployment/AdminDeploymentList.svelte [hunk 6]
```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	import type { GetDeploymentsQuery } from '$lib/graphql/_gen/typed-document-nodes';
+	import type { OperationStore } from '@urql/svelte';
+	import { displayFullName } from '../format';
```
Comment (Augustin82) [inline src/lib/ui/AdminDeployment/AdminDeploymentList.svelte:4]: nitpick : use $lib alias

File: src/lib/ui/AdminDeployment/AdminDeploymentList.svelte [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+	import LoaderIndicator from '../utils/LoaderIndicator.svelte';
```
Comment (Augustin82) [inline src/lib/ui/AdminDeployment/AdminDeploymentList.svelte:5]: nitpick : use $lib alias

File: src/routes/admin/__layout.svelte [hunk 8]
```diff
@@ -1,25 +1,39 @@
...
+				status: 302,
+				redirect,
+			};
+		}
+		return {};
+	}
 </script>
 
-<script lang="ts">
+<script type="ts">
```
Comment (Augustin82) [inline src/routes/admin/__layout.svelte:19]: issue : this should be "lang", right?

File: src/routes/admin/__layout.svelte [hunk 9]
```diff
@@ -1,25 +1,39 @@
-<script lang="ts" context="module">
-	import LayerCDB from '$lib/ui/LayerCDB.svelte';
+<script type="ts" context="module">
```
Comment (Augustin82) [inline src/routes/admin/__layout.svelte:1]: issue : this should be "lang", right?

File: src/routes/admin/deployment/[uuid].svelte [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+	export let deploymentId: string;
+	const getDeploymentStore = operationStore(GetDeploymentByIdDocument, { id: deploymentId });
+	query(getDeploymentStore);
+
+	$: deployment = $getDeploymentStore?.data?.deployment;
+	$: professional_aggregate = $getDeploymentStore?.data?.professional_aggregate;
+</script>
+
+<svelte:head>
+	<title>Deploiment {deployment?.label} - carnet de bord</title>
```
Comment (Augustin82) [inline src/routes/admin/deployment/[uuid].svelte:29]: issue : Deploiment => Déploiement
nitpick : carnet de bord => Carnet de bord
nitpick : use <Text> to handle null

File: src/routes/admin/index.svelte [hunk 11]
```diff
@@ -1,9 +1,24 @@
...
+	const deploymentsStore = operationStore(GetDeploymentsDocument);
+	query(deploymentsStore);
+
+	function onAddDeployementClick() {
+		openComponent.open({ component: AdminDeploymentCreateLayer });
+	}
 </script>
+
+<svelte:head>
+	<title>Gestion des déploiements - carnet de bord</title>
```
Comment (Augustin82) [inline src/routes/admin/index.svelte:19]: nitpick : carnet de bord => Carnet de bord

File: src/routes/manager/index.svelte [hunk 12]
```diff
@@ -0,0 +1 @@
+yo
```
Comment (Augustin82) [inline src/routes/manager/index.svelte:1]: issue : what should go there?

**Review by Augustin82 — CHANGES_REQUESTED** [review]

File: src/routes/admin/__layout.svelte (hunk 8 above)
Comment (lionelB) [reply src/routes/admin/__layout.svelte:19]: yup -_-'

File: src/routes/manager/index.svelte (hunk 12 above)
Comment (lionelB) [reply src/routes/manager/index.svelte:1]: nothing it was a test :)

File: src/lib/ui/AdminDeployment/AdminDeploymentDetail.svelte [hunk 13]
```diff
@@ -24,9 +24,9 @@
 </script>
 
 <h1 class="fr-h2">
-	Déploiement <span class="text-france-blue-500">{deployment?.label}</span>
+	Déploiement <span class="text-france-blue-500">{deployment?.label ?? ''}</span>
 </h1>
-<p>Référent: {deployment?.managers.map((item) => displayFullName(item)).join(', ')}</p>
+<p>Référent&nbs: {deployment?.managers.map((item) => displayFullName(item)).join(', ')}</p>
```
Comment (Augustin82) [inline src/lib/ui/AdminDeployment/AdminDeploymentDetail.svelte:29]: issue : `&nbsp;`

File: src/routes/admin/deployment/[uuid].svelte [hunk 14]
```diff
@@ -26,7 +26,7 @@
 </script>
 
 <svelte:head>
-	<title>Deploiment {deployment?.label} - carnet de bord</title>
+	<title>Deploiement {deployment?.label ?? ''} - Carnet de bord</title>
```
Comment (Augustin82) [inline src/routes/admin/deployment/[uuid].svelte:29]: nitpick : Déploiement

**Review by Augustin82 — APPROVED** [review]

**Pull request merged**